scipy/interpolate/tests/test_interpolate.py

```diff
@@ -833,52 +836,62 @@ def test_lagrange(self):
         assert_array_almost_equal(p.coeffs,pl.coeffs)
 
 
+@xfail_xp_backends("dask.array", reason="lacks nd fancy indexing")
+@xfail_xp_backends("jax.numpy", reason="immutable arrays")
+@xfail_xp_backends("array_api_strict", reason="fancy indexing __setitem__")
+@skip_xp_backends(cpu_only=True)
```

Comment:
It would be quite nice to use the `xp_capabilities` / `make_xp_skip` machinery now it's quite well established

Reply:
This is addressed now, thanks Jake!